### PR TITLE
feat(multiship): multiShip hook-getCheckout

### PR DESCRIPTION
**Pull request by kibo-geetanshuchhabra**

### CODE REVIEW

**Review by TriptiVermaXP — APPROVED** [review]

File: hooks/queries/multiShip/useCheckoutQueries/useCheckoutQueries.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { useQuery } from 'react-query'
+
+import { makeGraphQLClient } from '@/lib/gql/client'
+import { getMultiShipCheckoutQuery } from '@/lib/gql/queries'
+import { checkoutKeys } from '@/lib/react-query/queryKeys'
+
+import type { Checkout } from '@/lib/gql/types'
+interface UseMultiShipCheckout {
+  checkoutId?: string
```
Comment (kibo-sushant) [inline hooks/queries/multiShip/useCheckoutQueries/useCheckoutQueries.ts:9]: can checkoutId nullable?

File: hooks/queries/multiShip/useCheckoutQueries/useCheckoutQueries.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  initialCheckout?: Checkout
+}
+export interface UseMultiShipCheckoutResponse {
+  data: Checkout | undefined
+  isLoading: boolean
+  isSuccess: boolean
+}
+
+const getCheckout = async (checkoutId?: string | null) => {
```
Comment (kibo-sushant) [inline hooks/queries/multiShip/useCheckoutQueries/useCheckoutQueries.ts:18]: In what senario checkoutId will be null?

**Review by kibo-sushant — APPROVED** [review]

File: hooks/queries/multiShip/useCheckoutQueries/useCheckoutQueries.ts (hunk 1 above)
Comment (kibo-geetanshuchhabra) [reply hooks/queries/multiShip/useCheckoutQueries/useCheckoutQueries.ts:9]: Resolved

File: hooks/queries/multiShip/useCheckoutQueries/useCheckoutQueries.ts (hunk 2 above)
Comment (kibo-geetanshuchhabra) [reply hooks/queries/multiShip/useCheckoutQueries/useCheckoutQueries.ts:18]: Resolved

File: lib/gql/queries/checkout/get-multi-ship-checkout-query.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import {
```
Comment (kibo-kevinwatts) [inline lib/gql/queries/checkout/get-multi-ship-checkout-query.ts:1]: I think we should name this folder something "multiship-checkout" to keep them separate.
Also, can you work with @kibo-sahilarora and make sure we re-use the same fragments / properties from this
https://github.com/KiboSoftware/nextjs-storefront/pull/175/files#diff-1a5bace280e3a7b4dee09ef29699a18383763fdcc105a3d5d800123ee7489453

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]

File: lib/gql/queries/checkout/get-multi-ship-checkout-query.ts (hunk 3 above)
Comment (kibo-geetanshuchhabra) [reply lib/gql/queries/checkout/get-multi-ship-checkout-query.ts]: Resolved

**Pull request merged**